packages/test/src/integrationUtils.ts

```diff
@@ -26,13 +26,19 @@ interface CreateNewUser {
     email: string;
     isNew: boolean;
     message: string;
+    userId: number;
 }
 
 interface UserCredentials {
     password: string;
     username: string;
 }
 
+export interface TestUser extends UserCredentials {
```

Comment:
Would it make sense for TestUser to be a place to hang helpers that could (for example) get a requestContext for that user?